prefab.proto

```diff
@@ -38,31 +37,62 @@ message ConfigValue {
     bytes bytes = 3;
     double double = 4;
     bool bool = 5;
-    FeatureFlag feature_flag = 6;
+    WeightedValues weighted_values = 6;
```

Comment:
Is there a reason to use `WeightedValues` here instead of `repeated WeightedValue`? It looks like the `WeightedValues` message is just an abstraction over that that's only used in this one place.

Reply:
it's a proto thing. You can't have a `repeated` in a `oneof`   :(

Reply:
gotcha.